packages/host/app/components/ai-assistant/message/index.gts

```diff
@@ -230,6 +271,21 @@ export default class AiAssistantMessage extends Component<Signature> {
         {{/if}}
 
         <div class='content' data-test-ai-message-content>
+          {{#if @reasoningContent}}
+            <div class='reasoning-content'>
+              {{#if (eq 'Thinking...' @reasoningContent)}}
+                Thinking...
```

Comment:
```suggestion
                Thinking…
```

It’s subtle but it does render differently

<img width="106" alt="feb11 2025-03-11 09-45-52" src="https://github.com/user-attachments/assets/a552e746-8757-4bf1-9863-c02304de8de9" />

<img width="106" alt="feb11 2025-03-11 09-46-02" src="https://github.com/user-attachments/assets/adf58eee-7ed6-4484-acba-5edec803fb2f" />